### PR TITLE
Update the DB schema version directly rather than slowly, laboriously…

### PR DESCRIPTION
**Pull request by kmaglione**
… "faking" every migration.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -50,12 +50,15 @@ tdd:
 test_failed:
 	$(DOCKER_PREFIX) py.test --lf $(ARGS)
 
+HIGHEST_MIGRATION = $$(cd migrations; \
+		       ls | sort -rn | sed -n 's/^\([0-9]\+\).*/\1/p' | head -1)
```
Comment (magopian) [inline Makefile]: ewwww

Comment (magopian) [inline Makefile]: How about modifying schematic (yeah, I know) to add a parameter that provides the result of running https://github.com/mozilla/schematic/blob/master/schematic#L183?

Or even better, make schematic be a real package that can be installed (and not only a script), so we can `from schematic import find_upgrades`.

Comment (kmaglione) [inline Makefile]: My first thought was that I'd update schematic, but then I looked at schematic and this seemed like the cleaner option. But I guess I'll give it a try.

**Comment by andymckay** [comment]
I would really like us to move to using django-migrations and completely get rid of schematic. I think we shouldn't worry about altering this.

**Pull request closed without merging**